dictionary.json

```diff
@@ -128,6 +128,17 @@
       "is_array": true,
       "type": "device"
     },
+    "domain": {
```

Comment:
Is this being used anywhere? I only see `domains` not `domain` in the email object

Reply:
It's a bit nitpicky, but in the latest OCSF, the definition of `domain` was updated from `The name of the domain.` to `The name of the domain. See specific usage.`

The `See specific usage.` is a useful convention OCSF has adopted post v1 for more generic dictionary attributes whose definitions may vary depending on the specific class/object in which they are used. I updated this definition via our Splunk extension so it aligns with OCSF 1.4